sentry/sentry.conf.example.py

```diff
@@ -341,6 +341,14 @@ def get_internal_network():
 # https://django-csp.readthedocs.io/en/latest/configuration.html
 # CSP_SCRIPT_SRC += ["example.com"]
 
+############################
+# Sentry Endpoint Settings #
+############################
+
+# URI Prefixes for generating DSN URLs
+# (Defaults to URL_PREFIX by default)
+SENTRY_ENDPOINT = "https://sentry.ingest.example.com"
```

Comment:
The default should be commented. We let people uncomment this line if they need it.